server/index.js

```diff
@@ -1,14 +1,14 @@
-const express = require("express");
-const path = require("path");
+const express = require('express');
+const path = require('path');
 const app = express();
 const port = 3000;
 
 app.listen(port, function () {
   console.log(`http://localhost:${port}/ server is on`);
 });
 
-app.use(express.static(path.resolve(__dirname,"../dist")));
+app.use(express.static(path.resolve(__dirname, '../dist')));
 
-app.get("/", function (request, response) {
-  response.sendFile(path.resolve(__dirname, "../dist/index.html"));
-});
\ No newline at end of file
+app.get('/', function (request, response) {
```

Comment:
Для функций колбеков, обычно используются стрелочный функции
```suggestion
app.get('/', (request, response) => {
```